src/Components/map_progression/ClickModal.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import "./ClickModal.css";
+
+class ClickModal extends React.Component {
+    render(){
+        return (
+        <div classname="ClickName">
+        <div className="modal fade" id="exampleModal" tabIndex={-1} role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div className="modal-dialog" role="document">
+            <div className="modal-content">
+                <div className="modal-header">  
+                <button type="button" className="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">×</span>
+                </button>
+                </div>
+                <div className="modal-body">
+                Êtes-vous sûr(e) de vouloir faire ce quizz ?
+                </div>
+                <div className="modal-footer">
+                <button type="button" className="btn btn-secondary" data-dismiss="modal">Non</button>
+                <button type="button" className="btn btn-primary">Oui !</button>
+                </div>
+            </div>
+            </div>
+        </div>
+        </div>
```

Comment:
Fix indentation

Reply:
Your modal works using bootstrap and jquery. It manipulates the DOM which will surely break React at some point. Use bootstrap-react's modal component instead or build your own.